docs/walkthrough/working-with-elasticsearch.md

```diff
@@ -56,3 +56,77 @@ elastic.search(
     size=1000,
 )
 ```
+
+## Monitoring your Elasticsearch cluster
+
+It's helpful to periodically review the Quilt Elasticsearch cluster
+health to ensure performance is optimal and cost-effective.
+
+To review your Elasticsearch cluster:
```

Comment:
Here and elsewhere say OpenSearch as that's what AWS calls it now and that's how users will find it in console.

Reply:
The AWS service is called OpenSearch, but the clusters Quilt uses are still Elasticsearch (see the `Logical ID` value of the `Search` resource - it is still called `AWS::Elasticsearch::Domain`.

It makes sense to change references to the _AWS service_, but not the cluster ID, which is still Elasticsearch.